scripts/run-alphanet-parachain.sh

```diff
@@ -77,16 +77,22 @@ $PARACHAIN_BINARY \
   --rpc-port $((PARACHAIN_PORT + 10 + 1)) \
   --ws-port $((PARACHAIN_PORT + 10 + 2)) \
   --validator \
+  --rpc-cors all \
+  --rpc-methods=unsafe \
+  --execution wasm \
+  --wasm-execution compiled \
   --name parachain_$PARACHAIN_INDEX \
   $PARACHAIN_BASE_PATH \
-  '-linfo,evm=trace,ethereum=trace,rpc=trace' \
-  --chain $PARACHAIN_SPEC_PLAIN  \
+  '-linfo,evm=debug,ethereum=trace,rpc=trace,cumulus_collator=debug,txpool=debug' \
+  --alice  \
+  --chain staging \
```

Comment:
@JoshOrndorff I tried many ways to make it work but couldn't fine one (using dev, local, alphanet....) that doesn't complain about missing spec file.
Can you show me again how this is supposed to be done ?